### DISCUSSION TITLE
Accept donations using liberapay in the future

### DISCUSSION THREAD

**Discussion by Duckbilled** [Zettlr/Zettlr#2897] [Ideas]
I would love to donate, but rather just pay with a card instead of having an account on patreon.

I believe this is possible through liberapay. 

https://liberapay.com/
https://github.com/liberapay/liberapay.com

**Comment by nathanlesage** [comment]
In general a nice idea, and others have also voiced that. However, Germany is pretty strict in its tax laws, and since there is no incorporated organization behind Zettlr yet, I personally prefer Patreon, since they take care of the most important taxes already and I want to avoid having to deal with these issues as well.

**Reply by Duckbilled** [reply]
I understand, wasn’t aware of that. Thank you for replying!
